types/node/fs/promises.d.ts

```diff
@@ -1008,6 +1008,12 @@ declare module "fs/promises" {
             | (ObjectEncodingOptions & {
                 mode?: Mode | undefined;
                 flag?: OpenMode | undefined;
+                /**
+                 * If all data is successfully written to the file, and `flush`
+                 * is `true`, `filehandle.sync()` is used to flush the data.
+                 * @default false
+                 */
+                flush?: boolean;
```

Comment:
```suggestion
                flush?: boolean | undefined;
```
every other usage of flush uses undefined, same other props in this block

Reply:
Thanks, Peter.